### DISCUSSION TITLE
FlashIAPBlockDevice definitions

### DISCUSSION THREAD

**Discussion by robzed** [khoih-prog/FS_Nano33BLE#6] [Q&A] [answered]
What is this?

`  static FlashIAPBlockDevice wholeBD(FLASH_BASE, 0x80000);`

It looks like it isn't used, because this one is:

`  static FlashIAPBlockDevice bd(FLASH_BASE, (NANO33BLE_FS_SIZE_KB * 1024));`

Is it safe to have two of these definitions?

**Comment by khoih-prog** [comment, marked as answer]
Hi @robzed 

This is just a left-over and won't harm. Just a waste of memory.

I'll remove in the new release, mentioned in [**Half size of flash** #2](https://github.com/khoih-prog/FS_Nano33BLE/discussions/2)
